environment_setup/install_clion.sh

```diff
@@ -4,12 +4,56 @@ echo "================================================================"
 echo "Installing CLion"
 echo "================================================================"
 
-# Clion can be installed through snapcraft
-# https://snapcraft.io/clion
-sudo apt-get install snapcraft
-sudo snap install clion --classic
+clion_version="2019.2.2"
+usr_location="/usr/local/bin/clion"
+
+if [ -e ${usr_location} ]
```

Comment:
hmm, so I'd suggest adding a symlink from the clion binary to `/usr/local/bin/clion-2019.2`, then one from `/usr/local/bin/clion`, so that we check if `/usr/local/bin/clion-2019.2` exists here. This would be helpful when we update the CLion version in the future.

Reply:
Is there a point in having two links? Wouldn't checking if `/opt/clion-<version>` exists serve the same purpose?